Resources/Prototypes/Nyanotrasen/Entities/Markers/Spawners/Random/books.yml

```diff
@@ -55,6 +55,7 @@
       - BookEngineersHandbook
       - BookScientistsGuidebook
       - BookSecurity
+      - BookSpaceLaw
```

Comment:
doesnt this book not work

Reply:
I'm not sure. I just saw that it was added in the last merge

Reply:
I see. I'll comment it out when I fix conflicts. Maybe can be added back if we make our own then.

Reply:
also the conflict is just from material reclaimer being removed